lib/index.spec.ts

```diff
@@ -310,6 +310,220 @@ test('.evaluateObject() should evaluate a VALUES formula', async () => {
 	});
 });
 
+test('.evaluateObject() should return never for NEEDS formula', async () => {
```

Comment:
```suggestion
test('.evaluateObject() should return never for NEEDS formula, if it's full moon on a Sunday', async () => {
```
😉 
Tests are much easier to read when we understand the actual scenario we're testing